news/_posts/2016-11-28-prestonbot-reaches-stable-version.md

```diff
@@ -0,0 +1,96 @@
+
+PrestaShop want to be available for every people in the world, and probably in the universe :)
+One of the more precious contributions to do that are the ones related to the translation and the content.
+
+For instance, thanks to many contributors PrestaShop 1.7 is already available in 25 languages, making the software usable mostly everywhere in the world!
+ 
+The process to get a translation key available from Crowdin - the platform that host translations for every language of PrestaShop - to the users is still a little bit complex and we are improving the way we want to deliver the translation packs.
+
+PrestaShop 1.7 introduced the "domainization" of translation keys. Using the new back office of translations, this make the work of traductors. We are also faster on delivering new versions of translation packs in both `1.6` and `1.7` versions.
+
+### Make the core team management of translations easier
+
+What happens every time a contributor want to add or update a translation in PrestaShop?
+
+* We need to notify @AlexEven: our content manager, using *waiting for wording* label;
+* @AlexEven validate the contribution;
+* We merge the contribution;
+* We update the translation pack for every language;
+
+Most of the time, the update or the addition of a translation goes unnoticed by the team, often when happens in big contributions... until now.
+
+PrestonBot is able to analyze the contribution, and to add the **Waiting for wording** label when required. And @AlexEven only need to filter every contribution with the label, and to remove the label when the wording is valid.
+
+## Welcome and help the beginners
+
+Most of the time, people are realy afraid of contribute, because of the well-known "[Impostor syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome)".
+
+We - as a community - need to be friendly and kind, regarding the quality we expect from every contribution.
+We are looking to many ways to create a better environment to empower and encourage people to make their first contributor, because this is the most difficult one.
+
+This is why we have started to welcome community members in the core team, because we think everyone that want to be involved in this project is precious, and should be helped and thanked.
+
+In this case, the contribution of PrestonBot can help and the first use case we have adressed is to welcome every new contributor in our community.
+
+![Welcome drAlberT](http://i.imgur.com/MkAhH9d.png)
+
+As we do, we encourage you to welcome every new member and to be kind with every community member: 
+let's concentrate on what brings us closer, not on what differentiates us :)
+
+## We ❤ contributors
+
+PrestaShop can't really exists without the contributors.
+We are working hard to make your experience with GitHub easier for everyone: beginners, traductors, every contributor is welcomed.
```

Comment:
We are working hard to make the experience with GitHub easier for everyone **OR** We are working hard to make your experience with GitHub easier